grc/gui_qt/components/wiki_tab.py

```diff
@@ -40,29 +40,17 @@ def __init__(self, argv_enabled=False):
 
         self.setFloating(False)
 
-        active = None
-        if argv_enabled:
-            active = True
-        else:
-            if self.qsettings.value("appearance/display_wiki", False, type=bool) == True:
-                active = True
-            else:
-                active = False
-
-        if active:
-            try:
-                from qtpy.QtWebEngineWidgets import QWebEngineView
-                self.hidden = False
-            except ImportError:
-                log.error("PyQt QWebEngine missing!")
-                self.hide()
-                self.hidden = True
-                return
-        else:
+        try:
```

Comment:
The wiki widget causes a continuous stream of error messages on the console on Fedora+Wayland, so this was an opt-in feature until we figured that out. I'm ok with the saved setting, but we would just remove the `--wiki` arg and default to off in that case. No need to have a command line way to turn the feature on, but not turn it back off.

Reply:
I tested on F39 and got no errors. Sure, if save and restore the panel settings, there is no need for an command line option.
By the way , I just saw that the  Display Wiki Tab option can be modified in the preference editor in the Appearance Tab

Reply:
I'm running Wayland and the nvidia driver. It could be specific to that.